ui/src/App.js

```diff
@@ -54,6 +56,8 @@ const App = () => (
         addToPlaylistDialog: addToPlaylistDialogReducer,
         activity: activityReducer,
         settings: settingsReducer,
+        recentAlbumOrPlaylist: recentAlbumOrPlaylistReducer,
+        player: playerReducer,
```

Comment:
Why two new reducers? Can't this be achieved with just one? Or even better, add the new state to the existing `playerQueue` reducer?

Reply:
I will add both to the existing playerQueue reducer.